test/Microsoft.ML.Benchmarks/Helpers/BenchmarkHelpers.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Microsoft.ML.Benchmarks
+{
+    internal class BenchmarkHelpers
+    {
+        public static string RootPath = Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.Parent.Parent.Parent.Parent.Parent.FullName;
+        public static string TestDataPath =  Path.Combine(RootPath, @"test/data");
+    }
```

Comment:
since we have the helper class, we could put the logic for building the path here:

```cs
internal static string GetPath(string fileName) => Path.Combine(TestDataPath, fileName);
```

and every benchmark would just call it providing a file name + those fields would be private and when we change the logic for building the path we can do it in one place with single change